test/test_dao.py

```diff
@@ -457,13 +457,13 @@ def __init__(self, **kwargs):
 
 class TestDao(dao.BaseDao):
     def get_all_sites(self):
-        return test_sites.values()
+        return list(test_sites.values())
 
     def get_sites_by_codes(self, site_codes):
         return [test_sites[site_code] for site_code in site_codes]
 
     def get_all_variables(self):
-        return test_variables.values()
+        return list(test_variables.values())
```

Comment:
That is an iterator. Casting to list is "safer" but not needed all the times. It does not hurt to have it there though.